std_msgs/Header header

int8[] road_options
float64[] x
float64[] y

int8 VOID=-1
int8 LEFT=1
int8 RIGHT=2
int8 STRAIGHT=3
int8 LANEFOLLOW=4
int8 CHANGELANELEFT=5
int8 CHANGELANERIGHT=6